FritzFrog/detect_fritzfrog.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
+
```

Comment:
The script is more portable if you use

`#!/usr/bin/env bash`

Reply:
Got it. Conflict fixed.